tests/acceptance/features/apiSpaces/changeSpaces.feature

```diff
@@ -15,15 +15,40 @@ Feature: Change data of space
     And the administrator has given "Alice" the role "Space Admin" using the settings api
     And user "Alice" has created a space "Project Jupiter" of type "project" with quota "20"
     And user "Alice" has shared a space "Project Jupiter" with settings:
-      | shareWith | Brian    |
+      | shareWith | Brian  |
       | role      | editor |
     And user "Alice" has shared a space "Project Jupiter" with settings:
       | shareWith | Bob    |
       | role      | viewer |
     And using spaces DAV path
 
 
-  Scenario Outline: Only space admin user can change the name of a Space via the Graph API
+  Scenario: Only space admin user can change the name of a Space via the Graph API
+    When user "Alice" changes the name of the "Project Jupiter" space to "Project Death Star"
+    Then the HTTP status code should be "200"
+    And the JSON response should contain space called "Project Death Star" and match
```

Comment:
can you check if the above request gives the response that contains only one object or it gives array of objects in `values`. If only one object then we can simple use `And the JSON data of the response should match` without any computation. if so then apply to other scenarios as well.